packages-exp/messaging-exp/src/helpers/register.ts

```diff
@@ -64,6 +55,38 @@ const WindowMessagingFactory: InstanceFactory<'messaging-exp'> = (
   return messaging;
 };
 
+const WindowMessagingInternalFactory: InstanceFactory<'messaging-exp-internal'> =
```

Comment:
Looks like the name is `messaging-internal` in other places.

Reply:
good catch. The latest committed was pushed 😬